demos/exhibition-demo/src/playgrounds/CollectionsExample.java

```diff
@@ -0,0 +1,24 @@
+package playgrounds;
+
+import various.Point;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class CollectionsExample implements Playground {

```

Comment:
- This is the only playground class with a `Example` suffix in its name. Is this intentional?
- Some visitors will probably ask how maps would be visualized; maybe we should add a map playground as well.

Reply:
The `Example` is intentional. I can't just call it `Collections`, as it is already taken.